int64 id
ltlplanner/region region
